ITEM.tsv

```diff
@@ -22,11 +22,11 @@ ITEM_20150317_000021	$Recommended for those who desire a heavier and stronger sw
 ITEM_20150317_000022	$Sabre 
 ITEM_20150317_000023	$Primary weapon of the Cavalry, optimized for slashing. The guard on the hilt helps protect the user's hand. 
 ITEM_20150317_000024	$Shamshir 
-ITEM_20150317_000025	$The large curved crescent edge allows for a powerful slash. 
+ITEM_20150317_000025	$A blade that has a long, curved crescent. It dishes out heavy slashes.
```

Comment:
And take out 'very very' words to strengthen the sentence's overall meaning

$The blade's crescent edge allows it to dish out might slashes. (?)